src/core/Toast/Toast.test.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Toast } from './Toast';
+import { axeTest } from '../../utils/test';
+
+describe('props', () => {
+  describe('children', () => {
+    const ToastWithElementChild = (
+      <Toast headingText="Lorem ipsum dolor sit">
```

Comment:
It seems that `headingText` is an optional prop so there is no reason to include it in this test.

Reply:
Same applies to `className` and `ariaLiveMode` tests where `headingText` props are given.

Reply:
Fixed. removed those.